ShareBook/ShareBook.Api/Configuration/JWTConfig.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using ShareBook.Repository.Infra.CrossCutting.Identity;
+using ShareBook.Repository.Infra.CrossCutting.Identity.Configurations;
+using System;
+
+namespace ShareBook.Api.Configuration
+{
+    public class JWTConfig
+    {
+
+        public void RegisterJWT(IServiceCollection services, IConfiguration configuration)
+        {
+            var tokenConfigurations = configureToken(services, configuration);
+            var signingConfigurations = configureSigning(services);
+            configureAuth(services, signingConfigurations, tokenConfigurations);
+
+        }
+
+        private TokenConfigurations configureToken(IServiceCollection services, IConfiguration configuration)
```

Comment:
Ajuste as nomenclaturas para seguir os padrões de .Net:

DO use PascalCasing for class names and method names.
DO use camelCasing for method arguments and local variables.

Completo: http://www.dofactory.com/reference/csharp-coding-standards

Reply:
Beleza é que na empresa onde trabalho usamos camelCasing para método privados, o visual studio até reclamou. Já corrigido mano =)